presto-orc/src/main/java/com/facebook/presto/orc/OrcReaderOptions.java

```diff
@@ -25,10 +25,26 @@
     private final boolean zstdJniDecompressionEnabled;
     private final boolean mapNullKeysEnabled;
     private final boolean enableTimestampMicroPrecision;
+    // if the option is set to true,
```

Comment:
In Presto code base each line can be 100+ characters. So the comment seems like it is split to multiple lines. It can be reduced to 2 lines instead of 3.

Reply:
Yes, the limit is 180, i integrated it into one line